risc0/zkp/src/verify/mod.rs

```diff
@@ -258,13 +250,12 @@ pub fn verify<'a, H, C, F>(
     hal: &'a H,
     circuit: &C,
     seal: &'a [u32],
-    journal: &'a [u32],
-    check_code: F,
+    check_globals: F,
```

Comment:
We really need to support both checking globals and checking the code root.  I'm OK with the ZKVM temporarily ignoring the code root until we can add the build support to precompute the valid roots, but the recursion support will also need to check the code root, so I'd like to keep the API in place.